test/installation/src/puppeteer-typescript.spec.ts

```diff
@@ -17,7 +17,7 @@ import {execFile, readAsset} from './util.js';
   () => {
     configureSandbox({
       dependencies: ['@puppeteer/browsers', 'puppeteer-core', 'puppeteer'],
-      devDependencies: ['typescript@4.7.4', '@types/node@18.17.15'],
+      devDependencies: ['typescript@5.0.0', '@types/node@18.17.15'],
```

Comment:
please revert this change. Also, let's publish a separate PR to bump deps if it requires fix.

Reply:
let me check it locally